ark/devnet/knownWallets.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Why would this be here? This is explorer specific and not used for anything else.

Reply:
In general getting more info out also for other ppl to build on top of this "info" and have stuff in a common place form wallets from whoever, not just our lite clients and our crypto classes.

Reply:
This would be used on the wallets at least. Other third party projects could find it useful.